securedrop/journalist_templates/base.html

```diff
@@ -17,6 +17,8 @@
     <div id="logout">
       {% if g.user and g.user.is_admin %}
       <a href="{{ url_for('admin_index') }}">Admin</a> | 
+      {% else %}
+      <a href="{{ url_for('edit_account') }}">Edit Account</a>
```

Comment:
Add a `|` after "Edit Account" so there's a visual separator between it and "Logout".